integrations/traceloop.md

```diff
@@ -1,93 +0,0 @@
----
-layout: integration
-name: Traceloop
-description: Evaluate and monitor the quality of your LLM apps and agents
-authors:
-  - name: Traceloop Team
-    socials:
-      github: traceloop
-      twitter: traceloopdev
-pypi: https://pypi.org/project/traceloop-sdk/
-repo: https://github.com/traceloop/openllmetry
-type: Monitoring Tool
-report_issue: https://github.com/traceloop/openllmetry/issues
-logo: /logos/traceloop.png
-toc: true
----
```

Comment:
Let's not delete this and add `version: Haystack 2.0` to the frontmatter. 
Not sure if it's just a 1.x integration. We need to update the code example though

Reply:
done